tests/fixed/TralingCommaOnFunctions.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine;
+
+class TralingCommaOnFunctions
+{
+    public function a(int $arg): void
+    {
+    }
+
+    public function b(
+        int $arg
```

Comment:
Would've expected a trailing comma here, plus bumping to PHP 8 as minimum supported version 🤔

Reply:
Well, I resolved it with patches now since there's php 7 still being supported.

When min php req is bumped, we can apply the patch and submit it as a PR.  
But there has been some movement against bumping it https://github.com/doctrine/coding-standard/pull/226 (got "debumped") and I kinda don't want to try to re-bump it again  (https://github.com/doctrine/coding-standard/pull/127) since I've already did.